File: DTO/System/Account/Requests/PermissionModel.cs
namespace DTO.System.Account.Requests;

public class PermissionModel
{
    public RoleModel? Role { get; set; }
    public string? ControllerName { get; set; }
    public bool IsView { get; set; }
    public bool IsAdd { get; set; }
    public bool IsEdit { get; set; }
    public bool IsDelete { get; set; }
}

File: GUI/src/app/models/system/permission.model.ts
export interface PermissionModel {
  role: string | null;
  roleId: string | null;
  controllerName: string;
  isView: boolean;
  isAdd: boolean;
  isEdit: boolean;
  isDelete: boolean;
  isApprove: boolean;
  isStatistic: boolean;
}
